region/maridia/outer/Crab Hole.json

```diff
@@ -469,12 +490,13 @@
     {
       "link": [2, 1],
       "name": "G-Mode Morph IBJ",
-      "requires": [
-        {"comeInWithGMode": {
-          "fromNodes": [2, 3],
+      "entranceCondition": {
```

Comment:
A springball grav jump could be added

Reply:
Good call. I think I've missed several of these. I should look through Maridia again and see if there are more sometime